ch01.asciidoc

```diff
@@ -4,15 +4,20 @@
 
 === What is the Lightning Network?
```

Comment:
chapter 3 also has an intro chapter that explains lightning again on a slightly more technical level (c.f.: https://github.com/lnbook/lnbook/blob/develop/ch03.asciidoc ) is the redundancy an issue? I personally don't think so. However we should align the statements. For example you removed the "payment system" part which I forgot when writing chapter 3. Yet I think that by the end of the day this is THE crucial property and main reason why wo do all of that lightning stuff anyway.